docs/Extensions.md

```diff
@@ -1,6 +1,36 @@
 # Extensions
 
-## Custom XML Extensions
+## Contribution in package.json
+
+### Custom language
+
+If your vscode extension defines a custom language `myxml` in package.json:
+
+ ```json
+"contributes": {
+    "languages": [
+        {
+            "id": "myxml",
+            "extensions": [
+                ".myxml"
+            ]
+        }
+    ]
+```
+
+and you want that `myxml` language contributesto vscode XML language server, you need to declare it with `xmlLanguageParticipants`: 
```

Comment:
Instead of contributes to, maybe use the wording "uses", since the main point of adding the language to `xmlLanguageParticipants` is to make sure lemminx is used when in the file.

Reply:
fixed